### PR TITLE
remove network acceleration due to issues discovered

### PR DESCRIPTION
**Pull request by bryanro92**
### Which issue this PR addresses:

Fixes connection reset issues with egress lockdown

### What this PR does / why we need it:
sets accelerated networking on the gateway VM to false

### Test plan for issue:
deploy this code in full service dev rp, attempt to reproduce known egress issues, standard e2e

### Is there any documentation that needs to be updated for this PR?
maybe. pending additional testing results and response from azure networking team

### CODE REVIEW

File: deploy/gateway-production.json [hunk 1]
```diff
@@ -257,8 +257,7 @@
                             {
                                 "name": "gateway-vmss-nic",
                                 "properties": {
-                                    "primary": true,
```
Comment (gvanderpotte) [inline deploy/gateway-production.json:260]: what was this primary parameter used for ?

**Comment by bryanro92** [comment]
testing with accelerated networking disabled on gateway is going well so far, but maybe we wont even need this if underlying issues of accelerated networking feature can be resolved in a timely fashion

File: deploy/gateway-production.json (hunk 1 above)
Comment (hawkowl) [reply deploy/gateway-production.json:260]: I've put this back

**Review by bennerv — APPROVED** [review]
LGTM - but hold until ready to enable.

**Review by bennerv — APPROVED** [review]
LGTM

**Pull request merged**